src/shared/components/annotation/CivicCard.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import Collapse from 'react-collapse';
+import {If, Then, Else} from 'react-if';
+import DefaultTooltip from 'shared/components/DefaultTooltip';
+import {ICache} from "shared/lib/SimpleCache";
+import {ICivicGeneVariant} from "shared/model/Civic.ts";
+import "./styles/civicCard.scss";
+
+export interface ICivicCardProps {
+    title: string;
+    geneName: string;
+    geneDescription: string;
+    geneUrl: string;
+    variants: {[name: string]: ICivicGeneVariant};
+}
+
+export default class CivicCard extends React.Component<ICivicCardProps, {}>
+{
+    constructor()
+    {
+        super();
+    }
+
+    public generateVariants(variantMap:{[name: string]: ICivicGeneVariant})
+    {
+        const list:JSX.Element[] = [];
+        
+        if (variantMap) {
+            for (let name in variantMap) {
+                let variant = variantMap[name];
+                let entryTypes: string = '';
+                for (let evidenceType in variant.evidence) {
+                    entryTypes += evidenceType.toLowerCase() + ': ' + variant.evidence[evidenceType] +', '
+                }
+                entryTypes = entryTypes.slice(0, -2)+'.'
+                 
+                list.push(this.variantItem(variant.url, variant.name, entryTypes, variant.description));
+            }
+        } else {
+            list.push(this.variantItem());
+        }
+
+        return list;
+    }
+
+    public variantItem(url?:string, name?:string, entryTypes?:string, description?:string)
+    {
+        let result;
+        
+        if (url || name || entryTypes || description) {
+            result = (
+            <div className="civic-card-variant">
+                <div className="civic-card-variant-header">
+                    <span className="civic-card-variant-name"><a href={url} target="_blank">{name}</a></span>
+                    <span className="civic-card-variant-entry-types">{" Entries: "+entryTypes}</span>
```

Comment:
Here, string template literal is more preferable to string concatenation: `Entries: ${entryTypes}`